helm/app-operator/templates/ingress.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     - host: {{ include "resource.default.name" . }}.{{ .Values.Installation.V1.Kubernetes.API.Address }}
       http:
         paths:
-          - path: /
+          - path: /status
```

Comment:
One line fixes are the best fixes. ❤️

Ping @cokiengchiara so you don't miss out. 😄